ffi/harfbuzz/Cargo.toml

```diff
@@ -5,14 +5,14 @@
 [package]
 name = "icu_harfbuzz"
 description = "HarfBuzz glue code for ICU4X"
-version = "0.1.2"
-license = "Unicode-3.0"
+version = "0.6.0"
```

Comment:
Question: why `0.6` when it was previously `0.1`

Reply:
to align with the `harfbuzz` version

Reply:
I don't see why we should try to sync with the `harfbuzz` version; we could decide to make our own changes that are not synchronized with the `harfbuzz` version. Like, maybe we add support for one of the other harfbuzz crates.

Reply:
At the moment, `icu_harfbuzz` is an implementation of `harfbuzz-traits` 0.6.0.